docs/whats-new.md

```diff
@@ -1,5 +1,13 @@
 # What's New
 
+## TransformFeedback enhancements
+
+`Model` object constructor now accepts `varyings` (an array of vertex shader varyings) and `drawMode`, these will be used in creating `Program` object.
```

Comment:
Maybe the top-line item is that:

TransformFeedback is now supported directly by the `Model` class, it is no longer necessary to work with the `Program` class.

> , these will be used in creating `Program` object.
Maybe drop?